src/agenteval/cli.py

```diff
@@ -573,9 +589,20 @@ def eval_command(
 
     # Write the config portion of the results file
     os.makedirs(log_dir, exist_ok=True)
-    with open(os.path.join(log_dir, EVAL_FILENAME), "w", encoding="utf-8") as f:
-        unscored_eval_config = EvalConfig(suite_config=suite_config, split=split)
-        f.write(unscored_eval_config.model_dump_json(indent=2))
+    eval_config = EvalConfig(suite_config=suite_config, split=split)
+
+    eval_config_path = os.path.join(log_dir, EVAL_CONFIG_FILENAME)
+    if not os.path.exists(eval_config_path):
+        with open(eval_config_path, "w", encoding="utf-8") as f:
+            f.write(eval_config.model_dump_json(indent=2))
+    else:
+        with open(eval_config_path, "r", encoding="utf-8") as f:
+            existing_config = EvalConfig.model_validate_json(f.read())
+        if existing_config != eval_config:
```

Comment:
Check to make sure that the suite config has not changed between subsequent runs of `eval`